doc/release-notes.md

```diff
@@ -145,3 +521,11 @@ These release are considered obsolete. Old release notes can be found here:
 - [v0.11.0](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.11.0.md) released Jan/15/2015
 - [v0.10.x](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.10.0.md) released Sep/25/2014
 - [v0.9.x](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.9.0.md) released Mar/13/2014
+
+[DIP-0023]: https://github.com/dashpay/dips/blob/master/dip-0023.md
+[DIP-0029]: https://github.com/dashpay/dips/blob/master/dip-0029.md
```

Comment:
Should we add here DIP-0024 with a comment something like "to be published after release"?

Reply:
You probably mean DIP-0027 cause DIP-0024 is already published.

Reply:
This section is simply a map `[short_name]: url`. It's not rendered.